src/common.ts

```diff
@@ -362,6 +362,29 @@ export const minus = (arg1: string | number | TBig, arg2: string | number | TBig
   return new Big(arg1).minus(arg2)
 }
 
+interface NObject {
+  [key: string]: string | number | undefined | null | void
+  }
+  // 获取url中参数值
+export const getQuery = (str:string, a:string) =>{
+  str = str.split('?')[1]
+  if (!str) {
+    return {}
+  }
+  const obj = {} as NObject
```

Comment:
可以优化为，如果不存在 `number` 类型可以去掉（需要看 getQuery 是否会对类型进行转换）
```suggestion
  const obj = {} as Record<string, string | number | undefined>
```